startup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Export mmseqs path
+export PATH=/MMseqs2/build/bin/:$PATH
+
+# Install DeepMind's OpenMM patch
+patch -p0 -d /tools/miniconda/envs/ottf-openfold/lib/python3.7/site-packages/ < /openfold/lib/openmm.patch
+
+# Activate environment
+source activate openfold-env
+
+# Install openfold
+python setup.py install
```

Comment:
you probably want to run the installation process in the container itself instead of eating the cost (and danger of it failing) everytime you launch the container.

Reply:
the patch can also be applied during the container build.  It's also probably not necessary to use a virtual environment within the container since the container won't be doing anything other than running openfold